VideoWeb/VideoWeb.AcceptanceTests/Helpers/GetHearingRow.cs

```diff
@@ -117,7 +117,7 @@ private void GetRespondentRepresentatives()
 
         private void GetParticipantCount()
         {
-            _hearingRow.ParticipantCount = Int32.Parse( _browser.Driver.WaitUntilVisible(ClerkHearingListPage.ParticipantCount(_hearingId)).Text);
+            _hearingRow.ParticipantCount = _browser.Driver.WaitUntilVisible(ClerkHearingListPage.ParticipantCount(_hearingId)).Text;
```

Comment:
Please can you add Trim() to the end of this line, otherwise it will fail on Safari